.test-infra/jenkins/job_LoadTests_RuntimeTypeChecking_Python.groovy

```diff
@@ -0,0 +1,133 @@
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import LoadTestsBuilder as loadTestsBuilder
+import PhraseTriggeringPostCommitBuilder
+import InfluxDBCredentialsHelper
+
+def now = new Date().format("MMddHHmmss", TimeZone.getTimeZone('UTC'))
+
+def loadTestConfigurations = { datasetName ->
+  [
+    [
+      title          : 'Runtime Type Checking Python Load Test: On | Simple Type Hints',
+      test           : 'python -m apache_beam.testing.load_tests.runtime_type_check_on_test_py3',
+      runner         : CommonTestProperties.Runner.DATAFLOW,
+      pipelineOptions: [
+        job_name             : 'load-tests-python-dataflow-batch-runtime-type-check-1-' + now,
+        project              : 'apache-beam-testing',
+        region               : 'us-central1',
+        temp_location        : 'gs://temp-storage-for-perf-tests/loadtests',
+        publish_to_big_query : true,
+        metrics_dataset      : datasetName,
+        metrics_table        : 'python_dataflow_batch_runtime_type_check_1',
+        influx_measurement   : 'python_batch_runtime_type_check_1',
+        input_options        : '\'{"num_records": 0,"key_size": 0,"value_size": 0}\''
+        num_records          : 1000,
+        fanout               : 300,
+        num_workers          : 5,
+        autoscaling_algorithm: "NONE",
+        nested_typehint: 0  // False
+      ]
+    ],
+    [
+      title          : 'Runtime Type Checking Python Load Test: Off | Simple Type Hints',
+      test           : 'python -m apache_beam.testing.load_tests.runtime_type_check_off_test_py3',
+      runner         : CommonTestProperties.Runner.DATAFLOW,
+      pipelineOptions: [
+        job_name             : 'load-tests-python-dataflow-batch-runtime-type-check-2-' + now,
+        project              : 'apache-beam-testing',
+        region               : 'us-central1',
+        temp_location        : 'gs://temp-storage-for-perf-tests/loadtests',
+        publish_to_big_query : true,
+        metrics_dataset      : datasetName,
+        metrics_table        : 'python_dataflow_batch_runtime_type_check_2',
+        influx_measurement   : 'python_batch_runtime_type_check_2',
+        input_options        : '\'{"num_records": 0,"key_size": 0,"value_size": 0}\''
+        num_records          : 1000,
+        fanout               : 300,
+        num_workers          : 5,
+        autoscaling_algorithm: "NONE",
+        nested_typehint: 0  // False
+      ]
+    ],
+    [
+      title          : 'Runtime Type Checking Python Load Test: On | Nested Type Hints',
```

Comment:
Running load tests is a limited resource; could you test them both in the same pipeline? This should be sufficient for the purposes of noticing regressions.

Reply:
Yeah sure

I think the original goal of having both nested type hints and simple type hints as separate tests was to see if there was any performance difference between the two when runtime type checking was on, which would help us narrow down whether the performance drop came from the overhead of the decorator versus the actual type check itself, and also to test for the regressions separately.

I can merge them if it's okay with @udim